### PR TITLE
Update 170-baseline-your-database.mdx

### PR DESCRIPTION
**Pull request by KrunchMuffin**
Updated with a note about the known issue of file encoding for the initial migration

### CODE REVIEW

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/170-baseline-your-database.mdx [hunk 1]
```diff
@@ -212,6 +212,13 @@ The command will mark `0_init` as applied by adding it to the `_prisma_migration
 
 You now have a baseline for your current database schema. To make further changes to your database schema, you can update your Prisma schema and use `prisma migrate dev` to apply the changes to your database.
 
+[!NOTE]
+If you get the following error
+> Error: P3017
+> The migration 0_init could not be found. Please make sure that the migration exists, and that you included the whole name of the directory. (example: "20201207184859_initial_migration")
+
+You will need to change your migration.sql file encoding to UTF-8.
```
Comment (janpio) [inline content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/170-baseline-your-database.mdx:220]: Can you add a link to the issue here? I think that would be useful.

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/170-baseline-your-database.mdx [hunk 2]
```diff
@@ -212,6 +212,13 @@ The command will mark `0_init` as applied by adding it to the `_prisma_migration
 
 You now have a baseline for your current database schema. To make further changes to your database schema, you can update your Prisma schema and use `prisma migrate dev` to apply the changes to your database.
 
+[!NOTE]
```
Comment (janpio) [inline content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/170-baseline-your-database.mdx:215]: @ruheni Maybe you can suggest the needed "admonition" code here.

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/170-baseline-your-database.mdx (hunk 1 above)
Comment (ruheni) [inline content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/170-baseline-your-database.mdx:220]:
````suggestion

<Admonition>
If you get the following error, you will need to change your `migration.sql` file encoding to UTF-8:
```text
Error: P3017
The migration 0_init could not be found. Please make sure that the migration exists, and that you included the whole name of the directory. (example: "20201207184859_initial_migration")
```
</Admonition>
````

File: content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/170-baseline-your-database.mdx (hunk 2 above)
Comment (ruheni) [reply content/100-getting-started/02-setup-prisma/200-add-to-existing-project/110-relational-databases/170-baseline-your-database.mdx:215]: https://github.com/prisma/docs/pull/5387/files#r1372212071

**Pull request closed without merging**